manifest_staging/deployment/provider-azure-installer.yaml

```
@@ -47,6 +47,10 @@ spec:
             limits:
               cpu: 50m
               memory: 100Mi
+          ports:
+            - containerPort: {{ .Values.metricsAddr }}
```

```suggestion
            - containerPort: 8898
```

This is a static yaml